lib/kafka/server.go

```diff
@@ -2,85 +2,121 @@ package kafka
 
 import (
 	"encoding/json"
-	"time"
 	"log"
+	"time"
 
-	"github.com/zorkian/kafka"
+	freight "github.com/discordapp/discord/discord_freight"
```

Comment:
This is the wrong URL. In addition, `punt` is a public repo, and `discord` is private so we should not do this.